sampler/xray/lib/opentelemetry/sampler/xray/aws_xray_remote_sampler.rb

```diff
@@ -102,6 +121,8 @@ def retrieve_and_update_sampling_rules
           else
             OpenTelemetry.logger.error('GetSamplingRules Response is falsy')
           end
+        rescue StandardError => e
+          OpenTelemetry.logger.error("Error occurred when retrieving or updating Sampling Rules: #{e}")
```

Comment:
I forgot to point this out on the previous PR, but the [`OpenTelemetry.handle_error` API](https://open-telemetry.github.io/opentelemetry-ruby/opentelemetry-api/v1.5.0/OpenTelemetry.html#handle_error-instance_method) can be useful when rescuing errors. Users can define their own error handler to manage the exceptions. If they don't it'll just fall back to calling `OpenTelemetry.logger.error` with the message and first line of the backtrace.

Reply:
Thanks, I switched to [OpenTelemetry.handle_error](https://open-telemetry.github.io/opentelemetry-ruby/opentelemetry-api/v1.5.0/OpenTelemetry.html#handle_error-instance_method) for rescuing errors.